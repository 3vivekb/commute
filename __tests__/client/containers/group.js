/* global describe, expect, it */

import { mount } from 'enzyme'
import { mountToJson } from 'enzyme-to-json'
import React from 'react'
import { Provider } from 'react-redux'

import { makeMockStore, mockStoreData } from '../../../test-utils/mock-store.js'

import Group from '../../../client/containers/group'

const mockStore = makeMockStore(mockStoreData)

describe('Container > Group', () => {
  it('renders correctly', () => {
    // mount component
    const tree = mount(
      <Provider store={mockStore}>
        <Group />
      </Provider>
    )
    expect(mountToJson(tree.find(Group))).toMatchSnapshot()
  })
})
