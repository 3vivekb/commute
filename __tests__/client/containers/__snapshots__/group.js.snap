exports[`Container > Group renders correctly 1`] = `
<Connect(Group)>
  <Group
    dispatch={[Function]}
    user={Object {}}>
    <h1>
      Group
    </h1>
  </Group>
</Connect(Group)>
`;
